scim/resource_user_test.go

```diff
@@ -440,6 +442,62 @@ func TestResourceUserDelete_Error(t *testing.T) {
 	require.Error(t, err, err)
 }
 
+func TestResourceUserDeleteReposAndDirs(t *testing.T) {
```

Comment:
please add separate test invariants:
* where deleting repos fail
* where repos to be deleted is not found (assert actual user deletion is called).
* where deleting notebook dirs fail
* where notebook dirs to be deleted is not found (assert actual user deletion is called).